vue-components/src/standalone.js

```diff
@@ -0,0 +1,236 @@
+import { VtkWASMLoader, createFuture } from "./wasmLoader";
```

Comment:
This seems odd. why is standalone code here in trame-vtklocal? are you planning to move this out sometime later?

Reply:
Correct me if I'm wrong. From what i understnad, trame-vtklocal used `vtkRemoteSession` only, right? i think we should put this and code related to standalone into either vtk gitlab repo or the [kitware/vtk-wasm](https://github.com/Kitware/vtk-wasm) github repo.

Reply:
It is true that currently, the only widget in trame is using `vtkRemoteSession`, but that does not mean we could not have another one that ease the usage of JS/wasm within trame. Imagine a vscode editing a local JS file and have the result showing live on a preview section... 

Also, I didn't wanted to add overhead for setting up CI and a new npm project for it (for now). So it is there to get the benefit of the auto-publish to npm and so on. It rely on the same initialization module as the vtkRemoteSession, which allow me to not duplicate that code for now.